proseco/acq.py

```diff
@@ -598,14 +598,14 @@ def get_initial_catalog(self):
             # For include stars where the halfw is not going to be optimized
             # then then override the box size that was just found with the
             # user-supplied value.
-            for include_id, include_halfw in zip(self.include_ids, self.include_halfws):
-                if include_id not in self.include_optimize_halfw_ids:
-                    # Find the position in box_sizes that corresponds to include_id
-                    # and set to the specified include_halfw.
-                    for idx in range(len(acq_indices)):
-                        if include_id == cand_acqs[idx]['id']:
-                            box_sizes[idx] = include_halfw
-                            break
+            include_halfw_for_id = {iid: halfw for iid, halfw in zip(self.include_ids,
+                                                                     self.include_halfws)}
+
+            # We want the boxes to reference the same stars that the acq_indices index
+            for box_idx, acq_idx in enumerate(acq_indices):
+                if (cand_acqs[acq_idx]['id'] in self.include_ids and
+                        cand_acqs[acq_idx]['id'] not in self.include_optimize_halfw_ids):
+                    box_sizes[box_idx] = include_halfw_for_id[cand_acqs[acq_idx]['id']]
```

Comment:
You can probably find a cleaner way to do this @taldcroft , but it looked to me like the original logic here was just incompatible with 

https://github.com/sot/proseco/blob/78611dd7a0d5573f97b4753b4dfaf44ebc1f944f/proseco/acq.py#L624